common/src/main/java/org/jboss/hal/testsuite/creaper/command/AddSocketBinding.java

```diff
@@ -0,0 +1,156 @@
+package org.jboss.hal.testsuite.creaper.command;
+
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Jan Kasik <jkasik@redhat.com>
+ *         Created on 19.11.15.
+ */
+public class AddSocketBinding implements OnlineCommand {
```

Comment:
The class itself should have javadoc describing its intent.

BTW: this command is nicely done and I believe is good match for having it directly in creaper.
=> you should think about creating also merge request to https://github.com/wildfly-extras/creaper.